sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -60,6 +60,17 @@ class and wrapper class that allows lambda functions to be used as
 from apache_beam.utils import urns
 
 
+__all__ = [
+    'GetPValues',
```

Comment:
The only exported things from here should be PTransform and possibly label_from_callable. Maybe ptransform_fn, but mark it clearly as experimental (CloudML had some closure issues that were never fully resolved). The rest are internal implementation details.